example/index.tsx

```diff
@@ -36,6 +37,18 @@ const App = () => {
             <h2 style={{ color: 'darkcyan' }}>{children}</h2>
           ),
           bold: ({ children }) => <strong>{children}</strong>,
+          Asset: {
+            application: () => (
```

Comment:
Perhaps we should include the children here as an example?

Reply:
What do you mean by the children? Other mime-type groups?

`children` is not a passed as a prop to `Asset`.